draft-ietf-tls-external-psk-importer.md

```diff
@@ -87,20 +86,24 @@ when, and only when, they appear in all capitals, as shown here.
 
 # Overview
 
-Key importers mirror the concept of key exporters in TLS in that they diversify a key
-based on some contextual information before use in a connection. In contrast to key exporters,
-wherein differentiation is done via an explicit label and context string, the key importer
-defined herein diversifies external one PSK into one or more PSKs for use via a target protocol,
-KDF identifier, and optional context string. Additionally, the resulting PSK binder key is modified
-with a new derivation label to prevent confusion with non-imported PSKs.
-
-Imported keys do not require negotiation for use, as a client and server will not agree upon
-identities if not imported correctly. Endpoints may incrementally deploy PSK importer support
+The PSK Importer interface mirrors that of the TLS Exporters interface in that
+it diversify a key based on some contextual information. In contrast to the Exporters
+interface, wherein differentiation is done via an explicit label and context string,
+the PSK Importer interface defined herein takes an external PSK and identity and
+"imports" it into TLS, creating a set of "derived" PSKs and identities. Each of these
+derived PSKs are bound a target protocol, KDF identifier, and optional context string.
+Additionally, the resulting PSK binder keys are modified with a new derivation label
+to prevent confusion with non-imported PSKs.
+
+Imported keys do not require negotiation for use since a client and server will not agree upon
+identities if imported incorrectly. Endpoints may incrementally deploy PSK Importer support
 by offering non-imported keys for TLS versions prior to TLS 1.3. Non-imported and imported PSKs
 are distinct since their identities are different on the wire. See {{rollout}} for more details.
 
-Clients which import external keys TLS MUST NOT use these keys for any other purpose.
-Moreover, each external PSK MUST be associated with at most one hash function.
+Clients which import external keys MUST NOT use either the external keys or the derived
+keys for any other purpose. Moreover, each external PSK MUST be associated with at most
+one hash function, as per the rules in Section 4.2.11 from {{!RFC8446}}.
+See {{security-considerations}} for more discussion.
```

Comment:
@sayer this should address your comment. Please let me know if not!

Reply:
WFM